consensus/src/counters.rs

```diff
@@ -594,6 +594,26 @@ pub static NUM_TXNS_PER_BLOCK: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
+/// Traces node latency movement throughout the DAG
```

Comment:
Should we keep this in a separate file? within dag module so its cleaner? is this combined because we have some shared counters?

Reply:
I was sharing the round counter initially but probably separate is cleaner